src/main/java/com/microsoft/sqlserver/jdbc/IOBuffer.java

```diff
@@ -112,9 +115,17 @@
     static final byte FEDAUTH_INFO_ID_SPN = 0x02; // FedAuthInfoData is the SPN to use for acquiring fed auth token
 
     // AE constants
+    // 0x03 is for x_eFeatureExtensionId_Rcs
     static final int TDS_FEATURE_EXT_AE = 0x04;
     static final int MAX_SUPPORTED_TCE_VERSION = 0x01; // max version
     static final int CUSTOM_CIPHER_ALGORITHM_ID = 0; // max version
+    // 0x06 is for x_eFeatureExtensionId_LoginToken 
+    // 0x07 is for x_eFeatureExtensionId_ClientSideTelemetry 
+    // Data Classification constants
+    static final int TDS_FEATUREEXT_DATACLASSIFICATION = 0x09;
```

Comment:
Can we keep/make the naming consistent? There are 3 more similar constants:
`TDS_FEATURE_EXT_AE`, `TDS_FEATURE_EXT_FEDAUTH`, `TDS_FEATURE_EXTSION_ACK`. I would suggest using `TDS_FEATURE_EXT` for all of them.

Reply:
I would not rename others, but would make the one I added `TDS_FEATURE_EXT_DATACLASSIFICATION`